app/services/fetch_csv_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'net/http'
+require 'byebug'
+
+module Services
+  class FetchCsvService
```

Comment:
Isn't too repetitive the "service"?

Reply:
I think this can be done in two ways:

1. Keep module Services and remove Service at the end of the class FetchCsv.
2. Remove the module and change the way zeitwerk autolad paths from config.

This is available for all services defined in the app.